setup.py

```diff
@@ -29,7 +29,7 @@
         + extras_require['contrib']
         + extras_require['shellcomplete']
         + [
-            'pytest~=3.5',
+            'pytest>=6.0',
```

Comment:
```suggestion
            'pytest~=6.0',
```

Maybe we should just stay at `v6.X` so that we can properly test how changes to the API whenever `v7.0` comes out affects.